.github/workflows/check-whitespace.yml

```diff
@@ -0,0 +1,71 @@
+name: check-whitespace
+
+# Get the repo with the commits(+1) in the series.
+# Process `git log --check` output to extract just the check errors.
+# Add a comment to the pull request with the check errors.
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  check-whitespace:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set commit count
+      shell: bash
+      run: echo "::set-env name=CW_COMMITS::$((1+$COMMITS))"
+      env:
+        COMMITS: ${{ github.event.pull_request.commits }}
+
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: ${{ env.CW_COMMITS }}
```

Comment:
Oh, so the `CW_COMMITS` variable is needed because the `fetch-depth` field [cannot take arithmetic expressions](https://github.com/actions/checkout/blob/be6c44d969b1b004a9e0f7853e9cc9977ea0f7f0/src/input-helper.ts#L84-L89), which is a bummer.

But maybe use the name `FETCH_DEPTH` instead of `CW_COMMITS`, then?

Reply:
For the record, CW stood for `check whitespace` and not someone's name.